extension/background.ts

```diff
@@ -90,8 +90,12 @@ function initializeConfigFromCloudOrLocalStorageOrDefaults(cloudStorage) {
    * @param {boolean | number | string} defaultValue
    */
   function initConfig(key, defaultValue) {
-    let currentValue =
-      cloudStorage[key] || normalizeStringValue(localStorage[key]);
+    const cloudValue = cloudStorage[key];
```

Comment:
This definitely works but is a bit verbose:

Based on the idea from another user in the issue: https://github.com/melink14/rikaikun/issues/346#issuecomment-736203517

it seems like maybe it might be more idiomatic in typescript to just replace `||` with `??` (assuming it works as described)

Not sure if you already tried it but if not and it works, it seems cleaner!

(reference for nullish coalescing (`??`) operator in TS for FYI: https://medium.com/@sredmond/typescript-tip-of-the-week-nullish-coalescing-vs-logical-or-operator-72779807051)

Reply:
Yeah I hadn't initially tried it, but it does fix the issue that was occurring, so I updated the PR. That operator is so convenient.